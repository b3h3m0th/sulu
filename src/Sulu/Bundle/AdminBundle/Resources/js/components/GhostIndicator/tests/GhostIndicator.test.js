// @flow
import React from 'react';
import {render} from '@testing-library/react';
import GhostIndicator from '../GhostIndicator';

test('Should render with given locale', () => {
    const {container} = render(<GhostIndicator locale="de-at" />);
    expect(container).toMatchSnapshot();
});

test('Should render with given locale and className', () => {
    const {container} = render(<GhostIndicator className="test" locale="de-at" />);
    expect(container).toMatchSnapshot();
});
