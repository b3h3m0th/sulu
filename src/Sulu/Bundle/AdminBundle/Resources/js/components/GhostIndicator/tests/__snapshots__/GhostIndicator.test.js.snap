// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render with given locale 1`] = `
<div>
  <span
    class="ghostIndicator"
  >
    de-at
  </span>
</div>
`;

exports[`Should render with given locale and className 1`] = `
<div>
  <span
    class="ghostIndicator test"
  >
    de-at
  </span>
</div>
`;
